app/app.go

```diff
@@ -464,7 +464,7 @@ func NewApp(
 }
 
 func (app *AkashApp) registerUpgradeHandlers() {
-	app.keeper.upgrade.SetUpgradeHandler("akash_v0.13.0_cosmos_v0.43.0", func(ctx sdk.Context,
+	app.keeper.upgrade.SetUpgradeHandler("akash_v0.15.0_cosmos_v0.44.1", func(ctx sdk.Context,
```

Comment:
If this PR will be included in the `v0.15.0` release, then this change is fine. Also, we can actually combine multiple migrations for a module into one migration, and keep the `ConsensusVersion` for `deployment`,`escrow` & `market` at 2 (this PR changes that to 3).

But, if this PR is going in `v0.16` or later, then this change isn't valid. We will need to keep the existing upgrade handler as is, and register a new upgrade handler for this PR.